src/selectors/index.js

```diff
@@ -17,3 +19,113 @@ export function scheduledSectionsSelector(state) {
 export function selectedCourseNameSelector(state) {
   return state.getIn(['browse', 'selected', 'course', 'name']);
 }
+
```

Comment:
Can we move all of the helper functions to the top, then order the selectors below that by top level store fields? Like all the `search` selectors, then all the `browse` selectors, etc.

Reply:
Wouldn't hurt to delineate each of the three using comments either (see action-types.js)